fiware-region-sanity-tests/README.rst

```diff
@@ -23,6 +23,16 @@ Test environment
 __ `Python - Downloads`_
 __ `Vagrant - Downloads`_
 
+The E2E SNAT test uses DBus tech for communicating process. This one requires:
```

Comment:
I see no need for details... as E2E tests are part of the test suites, the following 3 items are just more prerequisites.